astroplan/constraints.py

```diff
@@ -938,6 +938,46 @@ def compute_constraint(self, times, observer=None, targets=None):
         return mask
 
 
+class NearMeridianConstraint(Constraint):
+    """
+    Constraint near the Meridian.
+    """
+    def __init__(self, min=None, boolean_constraint=True):
+        """
+        Parameters
+        ----------
+        min : `~astropy.units.Quantity` or `None`, optional
+            Minimum acceptable distance to meridian.
```

Comment:
```suggestion
            Minimum acceptable angular distance from the meridian.
```

Reply:
@ramonefoster Could you add this change too?

Reply:
I thought I did, but I didnt haha... anyways, I made the changes :)